libr/main/radare2.c

```diff
@@ -592,6 +607,12 @@ R_API int r_main_radare2(int argc, const char **argv) {
 			r_config_set (r->config, "dbg.profile", opt.arg);
 			break;
 		case 'R':
+			file = opt.arg;
```

Comment:
this looks wrong.
` -R [rr2rule] specify custom rarun2 directive`

`-R` is supposed to be used like `radare2 -R aslr=false -d /bin/ls`. It does not have a file as a argument. `-r` is probably what you wanted to check.

Reply:
truth is that now I'm not even sure if `-r` should be handled. The other two are OK as they're validating the input before appending it to the pre/post files list. I can't remember why I added `-R`, I think it's a leftover I forgot to delete.

Reply:
OK now I remember, and I think that the description, and the functionality of `-R` and `-r` is inverted. '-R' sets `customRarunProfile` which is later opened and appended to `dbg.profile` while, `-r` simply appends the given string to `dbg.profile`. Thoughts?

cc: @trufae

Reply:
So, `-R` creates a list of rarun2 directives, then it creates a temporary file, dump those entries in the temporary file and load dbg.profile with the temporary file.
`-r` directly gets a file and sets dbg.profile to that temporary file. So you should check whether the argument of `-r` is not an empty file.

Reply:
gotcha.